### DISCUSSION TITLE
Allow for toggling copilot on/off without notification

### DISCUSSION THREAD

**Discussion by miguelsolorio** [community/community#7605] [Copilot Conversations]
Currently when turning off copilot you are then asked whether to turn it off for the file or globally:

![image](https://user-images.githubusercontent.com/35271042/141359761-5bc7d091-0f4d-4dd0-a256-48d8c350f7aa.png)

![image](https://user-images.githubusercontent.com/35271042/141359711-e74449f9-13a6-4bfd-8f9c-4dbc54d2b2c1.png)

As a user who likes to toggle things on/off with a single command, I'd love to see a command to globally toggle copilot without the need to interact with a notification.

**Comment by dandavison** [comment]
I think https://github.com/github/feedback/discussions/7553#discussioncomment-1629050 may be a workaround for now

**Comment by JordanRickman** [comment]
**+1** from me, please provide this.
